model/registry/deprecated/network.yaml

```diff
@@ -35,12 +35,24 @@ groups:
         stability: experimental
         brief: Deprecated, use `server.port` on client spans and `client.port` on server spans.
         examples: [8080]
+      - id: peer.ip
+        type: string
+        deprecated: "Replaced by `net.sock.peer.addr`."
+        stability: experimental
+        brief: Deprecated, use `net.sock.peer.addr`.
```

Comment:
```suggestion
        deprecated: "Replaced by `network.peer.address`."
        stability: experimental
        brief: Deprecated, use `network.peer.address`.
```

Reply:
I was going off this and the PR it was deprecated in: https://github.com/open-telemetry/semantic-conventions/blob/main/schemas/1.25.0#L344

I don't see another entry in the schema for it?

Reply:
Seems the mapping wasn't possible to do automatically. I see the attributes now.